src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -193,6 +193,13 @@
                 .desc("Path to the bag config file")
                 .build());
 
+        configOptions.addOption(Option.builder("R")
+                .longOpt("repositoryRoot").argName("uri")
+                .hasArg(true).numberOfArgs(1).argName("uri")
+                .required(false)
+                .desc("Repository root URI")
```

Comment:
@peichman-umd :  Can you expand the description to include the intent of this feature, including whether or not it applies to both import and export modes as well as the default behavior when it is not specified?

Reply:
@dbernstein Opened new PR at #116